hail/python/test/hailtop/batch/test_batch.py

```diff
@@ -295,6 +292,8 @@ def test_resource_group_mentioned(self):
 class BatchTests(unittest.TestCase):
     def setUp(self):
         self.backend = ServiceBackend('test')
+        self.gcs_input_dir = 'gs://hail-services/batch-testing/resources'
+        self.gcs_output_dir = os.environ.get('SCRATCH').rstrip('/') + '/output'
```

Comment:
this one in particular, above moved to avoid encouraging global test state (which caused this issue) and because they were originally defined together and have a semantic connection